python/GafferImageUI/CatalogueUI.py

```diff
@@ -131,6 +135,81 @@
 
 )
 
+
+##########################################################################
+# Viewer hot-keys
+##########################################################################
+
+def addCatalogueHotkeys( editor ) :
+
+	if not isinstance( editor, GafferUI.Viewer ) :
+		return
+
+	editor.keyPressSignal().connect( _handleCatalogueHotkeys, scoped = False )
```

Comment:
Double underscore for private? While I'm nitpicking, our convention is to name slots to match the signals - so `__editorKeyPress` rather than `_handleCatalogueHotkeys`.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3387/commits/d3075a679d69fa049314d35d73e41737c7a154b6